Problem compiling micmac on macOS Mojave

make install -j4 on an iMac 27 gives me this:

/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:42:1: error: use of undeclared identifier 'cAppli_VByDate'
cAppli_VByDate::cAppli_VByDate(const std::string & aFullName, std::string & aOri):
^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:52:6: error: use of undeclared identifier 'cAppli_VByDate'
void cAppli_VByDate::CalcV(const std::string & aOut)
     ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:133:1: error: use of undeclared identifier 'cAppli_XifGps2Xml'
cAppli_XifGps2Xml::cAppli_XifGps2Xml(const std::string & aFullName,double aDefZ) :
^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:161:6: error: use of undeclared identifier 'cAppli_XifGps2Xml'
void cAppli_XifGps2Xml::ExportSys(cSysCoord * aSC,const std::string & anOri)
     ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:178:6: error: use of undeclared identifier 'cAppli_XifGps2Xml'
void cAppli_XifGps2Xml::ExportCoordTxtFile(std::string aOut, std::string aOutFormat)
     ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:218:1: error: use of undeclared identifier 'cAppli_XifDate'
cAppli_XifDate::cAppli_XifDate(const std::string & aFullName):
^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:228:6: error: use of undeclared identifier 'cAppli_XifDate'
void cAppli_XifDate::Export(const std::string & aOut)
     ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:239:21: error: use of undeclared identifier 'cAppli_XifDate'
std::vector<double> cAppli_XifDate::GetVDiffSecond()
                    ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:256:1: error: use of undeclared identifier 'cIm_XifGp'
cIm_XifGp::cIm_XifGp(const std::string & aName,cAppli_XifGps2Xml & anAppli) :
^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:256:48: error: unknown type name 'cAppli_XifGps2Xml'
cIm_XifGp::cIm_XifGp(const std::string & aName,cAppli_XifGps2Xml & anAppli) :
                                               ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:280:1: error: use of undeclared identifier 'cIm_XifDate'
cIm_XifDate::cIm_XifDate(const std::string & aName):
^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:287:1: error: use of undeclared identifier 'cIm_XifDate'
cIm_XifDate::cIm_XifDate(const std::string & aName, cElHour & aBeginTime):
^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:323:9: error: unknown type name 'cAppli_XifDate'
        cAppli_XifDate anAppli(aFullName);
        ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:328:9: error: unknown type name 'cAppli_VByDate'
        cAppli_VByDate anAppli(aFullName,aCalVOri);
        ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:359:5: error: unknown type name 'cAppli_XifGps2Xml'
    cAppli_XifGps2Xml anAppli(aFullName,aDefZ);
    ^
/Users/villavillacolle/micmac/src/uti_files/CPP_XifGps2Xml.cpp:405:5: error: unknown type name 'cAppli_XifGps2Xml'
    cAppli_XifGps2Xml anAppli(aFullName,aDefZ);

I previously built micmac successfully on another machine with Sierra. Any help is greatly appreciated, thank you!

Well, building the master branch like this gives the answer:

```
micmac/src/uti_files/CPP_XifGps2Xml.cpp:329:9: error: unknown type name 'cAppli_XifDate'
        cAppli_XifDate anAppli(aFullName);
        ^
micmac/src/uti_files/CPP_XifGps2Xml.cpp:334:9: error: unknown type name 'cAppli_VByDate'
        cAppli_VByDate anAppli(aFullName,aCalVOri);
        ^
micmac/src/uti_files/CPP_XifGps2Xml.cpp:365:5: error: unknown type name 'cAppli_XifGps2Xml'
    cAppli_XifGps2Xml anAppli(aFullName,aDefZ);
    ^
micmac/src/uti_files/CPP_XifGps2Xml.cpp:411:5: error: unknown type name 'cAppli_XifGps2Xml'
    cAppli_XifGps2Xml anAppli(aFullName,aDefZ);
```

--- a/src/TpMMPD/SimuBBA/simurolshut.h
+++ b/src/TpMMPD/SimuBBA/simurolshut.h
@@ -1,5 +1,0 @@
-#ifndef SIMUROLSHUT_H
-#define SIMUROLSHUT_H
-
-#endif // SIMUROLSHUT_H
-

--- a/src/uti_files/cpp_xifgps2xml.h
+++ b/src/uti_files/cpp_xifgps2xml.h
@@ -1,5 +1,0 @@
-#ifndef CPP_XIFGPS2XML_H
-#define CPP_XIFGPS2XML_H
-
-#endif // CPP_XIFGPS2XML_H
-

